webapp/sources/scala-ldap/src/main/scala/com/normation/ldap/sdk/LDAPConnectionProvider.scala

```diff
@@ -239,8 +239,10 @@ trait PooledConnectionProvider[LDAP <: RoLDAPConnection] extends LDAPConnectionP
     new LDAPConnectionPool(self.newUnboundidConnection, poolSize)
   } catch {
     case ex: LDAPException =>
-      LDAPConnectionLogger.error(s"Error during LDAP connection pool initialisation. Exception: ${ex.getDiagnosticMessage}")
-      throw new Error(ex.getDiagnosticMessage)
+      val msg = s"Error during LDAP connection pool initialisation. Exception: " +
+                s"${ex.getClass.getSimpleName}:${ex.getMessage}; Details: ${ex.getDiagnosticMessage}"
+      LDAPConnectionLogger.error(msg)
+      throw new Error(msg)
```

Comment:
shouldn't it be InitError?

Reply:
no, here we can use `Error`, init error is used in the test for active LDAP connection.